setupflaskdocker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#setup flaks docker
+cd ./flaskapp
+docker build -t flask:latest .
+cd ..
+#setup react docker
+cd ./reactapp
+docker build -t react:latest .
+cd ..
+#launch docker
+docker-compose up
+#pull stuff for flask
+docker exec -it elasticsearch /usr/share/elasticsearch/bin/elasticsearch-reset-password -b -u elastic
```

Comment:
Will there still be a manual step putting the password into the elasticsearch?

Reply:
the -b flag makes it so it doesn't require interaction. if you want i can have the output sent to passwords.txt

Reply:
I think it'd be cleaner. What I'd really like is to make it so that elastic.py can then read the password out of passwords.txt, but I'm not sure how to trigger that since elastic.py is getting run before this command goes off.